etl-job/fhir_import_export.py

```diff
@@ -241,7 +312,7 @@ def _main():
         raise Exception(f"unknown method {method}")
 
     # note, only the last output (a line in stdout with `[out]` prefix) is returned to the caller
-    print(f"[out] {json.dumps(output, separators=(',', ':'))}")
+    # print(f"[out] {json.dumps(output, separators=(',', ':'))}")
```

Comment:
gen3_util jobs reads this,  I think it would be better to continue to return it to the client

Reply:
this line was causing a python error. Will investigate this further on Monday.